### ISSUE TITLE
Vision 분야에 관한 내용 보충도 해주신다면 정말 감사하겠습니다

### ISSUE THREAD

**Issue by SangwooJ** [teddylee777/machine-learning#28]
먼저 훌륭한 repo를 만들어 주셔서 정말 감사합니다
 Vision 부분의 topic 이 추가된다면 좋을것 같습니다. 
vision 분야에서 무엇 부터 해야할지 저처럼 어려움을 겪는분들에게도 도움이될것같습니다

**Comment by teddylee777** [comment]
네 앞으로 Vision 분야도 적극적으로 Topic 추가해 보도록 하겠습니다. 혹시 공유해 주실 만한 좋은 repo나 글, 영상들을 pull request 해주시면 적극 반영하도록 하겠습니다. 감사합니다.
